src/main/java/frc/robot/commands/AimShoot.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
LOVE the comment block at the top. I THINK if you put an extra asterisk '*' after the one in this line and then put this whole block right above the constructor (or maybe the class definition) it will pop up when hovering over the command when using it.